### PR TITLE
update tiflash doc add dt_compression_method and dt_compression_level config

### PR DESCRIPTION
**Pull request by hehechen**
### What is changed, added or deleted? (Required)

Add `dt_compression_method` and `dt_compression_level` config in TiFlash doc.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8611
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -192,6 +192,11 @@ delta_index_cache_size = 0
     batch_cop_pool_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
+    # Compression algorithm of the TiFlash storage engine, supports LZ4, zstd, and LZ4HC, and is not case-sensitive. The LZ4 algorithm is used by default.
+    dt_compression_method = "LZ4"
+    # Compression level of the TiFlash storage engine, the default is 1. If dt_compression_method is LZ4, it is recommended to be 1; if dt_compression_method is zstd, it is recommended to be -1 or 1; if dt_compression_method is LZ4HC, it is recommended to be 9.
+    dt_compression_level = 1
+
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:199]:
```suggestion
    # Compression algorithm of the TiFlash storage engine. The value can be LZ4, zstd, or LZ4HC, and is case-insensitive. By default, LZ4 is used.
    dt_compression_method = "LZ4"
    # Compression level of the TiFlash storage engine. The default is 1. It is recommended that you set this value to 1 if dt_compression_method is LZ4, -1 or 1 if dt_compression_method is zstd, and 9 if dt_compression_method is LZ4HC.
    dt_compression_level = 1
```

**Review by shichun-0415 — APPROVED** [review]

**Review by flowbehappy — APPROVED** [review]

File: tiflash/tiflash-command-line-flags.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ This document introduces the command-line flags that you can use when you launch
...
     - `--algorithm`: The hash algorithm used for data validation. The value options are `xxh3` (default), `city128`, `crc32`, `crc64`, and `none`. This parameter is effective only when `version` is `2`.
     - `--frame`: The size of the validation frame. The default value is `1048576`. This parameter is effective only when `version` is `2`.
-    - `--compression`: The target compression algorithm. The value options are `lz4` (default), `lz4hc`, `zstd`, and `none`.
-    - `--level`: The target compression level. The default value is `-1`, which means automatic mode. The value range varies with different compression algorithms.
+    - `--compression`: The target compression algorithm. The value options are `LZ4` (default), `LZ4HC`, `zstd`, and `none`.
+    - `--level`: The target compression level. If not specified, the recommended compression level is automatically used by default according to the compression algorithm. If compression is set to LZ4, the default level is 1; if compression is set to zstd, the default level is 1; if compression is set to LZ4HC, the default level is 9. The value range varies with different compression algorithms.
```
Comment (shichun-0415) [inline tiflash/tiflash-command-line-flags.md:33]:
```suggestion
    - `--level`: The target compression level. If not specified, the recommended compression level is used by default according to the compression algorithm. If `compression` is set to `LZ4`, the default level is 1. If `compression` is set to `zstd`, the default level is 1. If `compression` is set to `LZ4HC`, the default level is 9. 
```

**Review by shichun-0415 — APPROVED** [review]
rest LGTM

File: tiflash/tiflash-command-line-flags.md [hunk 3]
```diff
@@ -29,8 +29,8 @@ This document introduces the command-line flags that you can use when you launch
...
     - `--algorithm`: The hash algorithm used for data validation. The value options are `xxh3` (default), `city128`, `crc32`, `crc64`, and `none`. This parameter is effective only when `version` is `2`.
     - `--frame`: The size of the validation frame. The default value is `1048576`. This parameter is effective only when `version` is `2`.
-    - `--compression`: The target compression algorithm. The value options are `lz4` (default), `lz4hc`, `zstd`, and `none`.
-    - `--level`: The target compression level. The default value is `-1`, which means automatic mode. The value range varies with different compression algorithms.
+    - `--compression`: The target compression algorithm. The value options are `LZ4` (default), `LZ4HC`, `zstd`, and `none`.
+    - `--level`: The target compression level. If not specified, the recommended compression level is used by default according to the compression algorithm. If `compression` is set to `LZ4`, the default level is 1. If `compression` is set to `zstd`, the default level is 1. If `compression` is set to `LZ4HC`, the default level is 9.
```
Comment (shichun-0415) [inline tiflash/tiflash-command-line-flags.md:33]:
```suggestion
    - `--level`: The target compression level. If not specified, the recommended compression level is used by default according to the compression algorithm. If `compression` is set to `LZ4` or `zstd`, the default level is 1. If `compression` is set to `LZ4HC`, the default level is 9. 
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -192,6 +192,10 @@ delta_index_cache_size = 0
     batch_cop_pool_size = 0
     ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
     enable_elastic_threadpool = false
+    # Compression algorithm of the TiFlash storage engine. The value can be LZ4, zstd, or LZ4HC, and is case-insensitive. By default, LZ4 is used.
+    dt_compression_method = "LZ4"
+    # Compression level of the TiFlash storage engine. The default is 1. It is recommended that you set this value to 1 if dt_compression_method is LZ4, -1 or 1 if dt_compression_method is zstd, and 9 if dt_compression_method is LZ4HC.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:197]:
```suggestion
    # Compression level of the TiFlash storage engine. The default value is 1. It is recommended that you set this value to 1 if dt_compression_method is LZ4, -1 (smaller compression rate, but better read performance) or 1 if dt_compression_method is zstd, and 9 if dt_compression_method is LZ4HC.
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**